sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -20,9 +20,9 @@ INSERT INTO `conditions` (`SourceTypeOrReferenceId`,`SourceGroup`,`SourceEntry`,
 (15,10908,0,0,0,9,0,24535,0,0,0,0,0,'','Show gossip option if 24535 is incomplete'),
 (15,10908,0,0,1,9,0,24563,0,0,0,0,0,'','Show gossip option if 24563 is incomplete');
 
-DELETE FROM `gameobject` WHERE `id`=201718;
+DELETE FROM `gameobject` WHERE `guid`=9004;
```

Comment:
please delete by guid AND id, just to make sure we are deleting exactly what we want and not some other gameobject